README.md

```diff
@@ -239,6 +239,62 @@ If there isn't, you'll need to create a local identifier source to generate "fak
 To develop on Microfrontends, you'll need to do some set-up. Follow the instructions in the
 [PIH SPA Frontend README](https://github.com/PIH/spa-frontend#pih-emr-spa-frontend).
 
+# Updating the Configuration for your Distribution
+
+ We are in the process of moving as much "configuration" out of the main PIH EMR code base and into distribution-specific files.  These files can be updated without updating the main PIH EMR code base.
+
+To do this, you'll need to work with two key PIH repos.  First, the "openmrs-config-pihemr" repo:
+
+https://github.com/PIH/openmrs-config-pihemr
+
+This is the "parent" configuration, that contains all configuration shared across all PIH EMR distributions.
+
+Then there is the specific "child" configuration for each distribution.  See Step 3 in "setting up a dev environment" above for the list of repos.
+
+When you make changes to the "child" repo, you need to compile the changes and then "deploy" them to the SDK server you are working on.  This can be done with the following command:
+
+```
+mvn clean compile -DserverId=[serverId]
+```
+
+When you make changes to the "parent" repo, you need to compile that project and install it to your local Maven repo, and then compile and deploy the child repo.  So from the parent project you would run:
+
+```
+mvn clean install
+```
+
+And then once that is complete, compile and "deploy" the child project locally:
+
+```
+mvn clean compile -DserverId=[serverId]
+```
+
+Note if you make changes to metadata installed via Initializer, you will need to restart your server to pick up the changes.  However, HTML Forms should be available to be "hot" reloaded... once you run the mvn commands above, doing a "reload" of a page should reload the form with your changes.
+
+Also note that if you commit any changes to either the parent or child config property, our CI server should immediately push them out to the relevant staging servers and restart OpenMRS, so, all going well, your changes should be up on the staging servers within 10-15 minutes of pushing your changes.
+
+# Updating the PIH EMR Code
+
+Beyond updating the configuration for your specific distribution you will also want to update the core PIH EMR code, which you can do as follows:
+
+If you are watching any modules, first execute "mvn openmrs-sdk:pull" to pull in any changes to these modules via git.
+
+Then, from the base directory of the mirebalais project run the following two commands to update any changes to modules
+you aren't watching:
+
+```
+$ git pull
+$ mvn openmrs-sdk:deploy -Ddistro=api/src/main/resources/openmrs-distro.properties -U
+```
+
+(I have created a shell script shortcut to execute the two commands above, pihemrDeploy.sh)
```

Comment:
```suggestion
Note that some of these steps will be repeated often enough that they warrant creating shortcuts. See
[Making things easy](https://github.com/PIH/openmrs-module-mirebalais#making-things-easy)
```

Reply:
I like keeping the reference to "pihemrDeploy.sh" in-line in this section, but I'll make a tweak to include your new text and link as well.